src/AWS.Deploy.Common/Recipes/Validation/OptionSettingItemValidatorList.cs

```diff
@@ -16,6 +16,10 @@ public enum OptionSettingItemValidatorList
         /// <summary>
         /// Must be paired with <see cref="RequiredValidator"/>
         /// </summary>
-        Required
+        Required,
+        /// <summary>
+        /// Must be paired with <see cref="FileValidator"/>
+        /// </summary>
+        File
```

Comment:
I ended up switching to `FileExists`, since I like having it suggest what scenario is being validated as opposed to what type of resource it's validating, but let me know if you prefer something else still.